src/com/owncloud/android/ui/activity/FileActivity.java

```diff
@@ -356,7 +365,15 @@ public OperationsServiceBinder getOperationsServiceBinder() {
     protected ServiceConnection newTransferenceServiceConnection() {
         return null;
     }
-    
+
+    @Override
+    public void onAccountsUpdated(Account[] accounts) {
+        // detect a change in the list of accounts
+        Log_OC.d(TAG, "onAccountsUpdated");
+        mDownloaderBinder.reviewDownloads();
```

Comment:
Check that ```mDownloaderBinder``` and ```mUploaderBinder``` are not null before calling methods in them; we don't know how the life cycle of binding and listening for the accounts are intertwined.